### ISSUE TITLE
[feature request]: Option for horizontal scrolling

### ISSUE THREAD

**Issue by emanruse** [newsboat/newsboat#1362]
On small screens it would be very useful to be able to quick scroll the article list to the right (and back). Some feeds have really long titles and opening each one sometimes takes too much time.

**Comment by Minoru** [comment]
> opening each one sometimes takes too much time.

Scrolling wouldn't be any faster: it's at least two keypresses, same as opening the article and quitting it.

I'm also not sure if STFL, our UI library, even supports it.

I'll accept a PR implementing this, though. It can probably add the same to Podboat #230 ;)

In the meantime, whenever you bump into a feed with long titles, you can just open one of the articles and use J/K (`next`/`prev`) to go between articles; it'll save you some typing.

**Comment by emanruse** [comment]
> Scrolling wouldn't be any faster: it's at least two keypresses, same as opening the article and quitting it.

Not really. From usability viewpoint it is actually quite different, because opening an article results in a new, completely separate, UI layout, in which one no longer sees the article list. The two UIs serve different intent: skimming through the article list (titles that may catch attention for further reading) is different from diving into already chosen content, when the rest of the articles are intentionally excluded from one's attention. Timing for both is different too. Eye fatigue is different too.

Think about journalctl and how you can right-scroll (one keypress) if lines are too long. This is how I imagine it working.

> I'll accept a PR implementing this, though. It can probably add the same to Podboat #230 ;)

Great.

> In the meantime, whenever you bump into a feed with long titles, you can just open one of the articles and use J/K (`next`/`prev`) to go between articles; it'll save you some typing.

Thanks. I know that, but it is different as explained above ;)
